src/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights/Configuration/ApplicationInsightsOutputConfiguration.cs

```diff
@@ -10,5 +10,6 @@ namespace Microsoft.Diagnostics.EventFlow.Configuration
     public class ApplicationInsightsOutputConfiguration: ItemConfiguration
     {
         public string InstrumentationKey { get; set; }
+        public string ConfigurationFilePath { get; set; }
```

Comment:
are you going to add the description for this addition in read.md and explain the format for the file?